db/seeds.rb

```diff
@@ -13,3 +13,11 @@
 unless LeadProvider.first
   LeadProvider.create!(name: "Test Lead Provider")
 end
+
+unless AdminProfile.first
```

Comment:
This is great, exactly what I was thinking of. We potentially want this as 
```
unless AdminProfile.first  || Rails.env.production?
```
however